.github/workflows/pr-go-workspace-check.yml

```diff
@@ -46,4 +46,4 @@ jobs:
           git diff
           echo "Please run 'make build-go' and commit the changes."
           exit 1
-        fi
\ No newline at end of file
+        fi
```

Comment:
not sure what changed here - whitespace?

Reply:
newline at the end of the file it seems

Reply:
there was previously no EOL - my editor added it, because EOLs at EOFs are good 😉